_posts/en/newsletters/2020-01-22-newsletter.md

```diff
@@ -0,0 +1,117 @@
+  digital signature, which the winning party can use to claim their
+  funds.  The oracle doesn't need to know the terms of the contract (or
+  even that there is a contract).  The contract can look like either the
+  onchain part of an LN transaction or it can be executed within an LN
+  channel.  This makes DLCs more private and efficient than other known
+  oracle-based contract methods, and it's arguably more secure as an
+  oracle who commits to a false result generates clear evidence of their
+  fraud.
+
+    This week Chris Stewart [announced][stewart dlc] that several
+    developers are working on a specification for using DLCs, with the
+    goal of creating a interoperable design for use between different
+    software, including LN implementations.  See their
+    [repository][dlcspecs] for the current specification.  Anyone
+    interested in DLCs may also wish to review the [scriptless
+    scripts][scriptless scripts examples] repository that documents
+    other clever applications of digital signature schemes to contract
+    protocols.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **River Financial launches using variety of Bitcoin tech:** In a [Twitter thread][river twitter thread]
+  and a [blog post][river blog introduction] introducing their platform, River
+  Financial announced their use of PSBT, script descriptors, LN, and native
+  segwit addresses by default.
+
+- **Wasabi notes RBF on incoming transactions:** Wasabi's incoming transaction
+  notification now [notes RBF signaling][wasabi rbf notification] and
+  whether the transaction is a replacement transaction. To enchance privacy,
+  Wasabi also randomly [signals RBF on 2% of transactions][wasabi rbf signaling].
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #17843][] helps resolve a discrepancy between how reused
+  UTXOs are determined in the `listunspent` and `getbalances` RPCs in
+  wallets using [reuse avoidance][Bitcoin Core #13756].
+
+- [Eclair #1247][] fixes the Sphinx privacy leak described in
+  [Newsletter #72][news72 sphinx] where a routing node might be able to
+  deduce a lower bound for the length of a path back to the source node.
+
+- [Eclair #1283][] allows [multipath payments][topic multipath payments]
+  (MPP) to traverse unannounced channels, which is necessary for
+  eclair-mobile to be able to make MPP payments.
+
+- [LND #3900][] allows a spender to send custom data records along
+  with their payment.  Using `lncli`, a user can pass the `--data` flag
+  along with the record ID and the data in hex, e.g. `65536=c0deba11ad`.
```

Comment:
Where does this example come from?

Reply:
It's a variation on https://github.com/lightningnetwork/lnd/pull/3900/commits/d978b55701472f8b33e566976c6df19489c2bcd9#diff-e43410ad1ea45c0f0bd96f9bef29a767R2099